app/controllers/auth.php

```diff
@@ -395,13 +396,14 @@ function ($email, $password, $success, $failure) use ($response, $request, $proj
     ->label('sdk.description', '/docs/references/auth/login-oauth.md')
     ->label('sdk.location', true)
     ->label('sdk.cookies', true)
-    ->label('abuse-limit', 50)
-    ->label('abuse-key', 'ip:{ip}')
+    // ->label('abuse-limit', 100)
```

Comment:
Please note we should uncomment and keep lines 399-400
Please note we should uncomment and keep lines 457-458

Lines: 438-439 can be deleted